### ISSUE TITLE
Voting (A) Shares Holder Requirements

### ISSUE THREAD

**Issue by DanM3rcurius** [SecretDecks/Documentation#127]
## Voting (A) Shares 
Shares that vote on executive decisions like roadmap or long term strategy and approvals like yearly book keeping reports.
Having Voting (A) Shares separate from Dividend (B) Shares and Veto (C) Shares allows for Steward-Ownership of an entity, as there can't be much conflict of interest. 

### Voting (A) Shares Requirements
**Can YOU *get* steward-ownership?**
All the below:
1) Be Core-Member for at least 6 months
2) You want to!
3) absolve Workshop (what does it mean to have steward-ownership in SecretDecks)
4) absolve Coaching (communication without violence)

**Can YOU *keep* steward-ownership?**
at least two of the below:
1) as freelancer work on average 20h/week
2) be employed full or part time by the DisCo
3) occupy an executive position in the DisCo

### Thoughts 
In a DisCo all power should be distributed equally to all but based upon merit. Hence executive decisions also need to be done collectively - I guess - (eventually or from the start?). 
#### Idea?
We could create custom voting tokens that are being distributed (from the 99% voting (A) shares) according to the overall amount of impact, skill/expertise, reputation built and time invested in the coop and on every (major/executive) voting decision anew. After voting the participants votes are recorded in their personal datasets and the voting tokens are burned **(This requires Hydra[head]technology on Cardano.**
#### Proposal for now?
We can start with one or two steward-ownerships then add more as we go.

99% of SecretDecks UG/GmbH would be held by SecretDecks GbR (change of shareholder doesn't require notary sign off!) all core members of SecretDecks can then exact executive voting power through the GbR. 
For non executive business decisions (like design or artist curation) members can still exact voting power (ideally the voting is weighted merit base on non exec levels as well)
 
**IMPORTANT:** Distinction between *steward-owner* and *member*! 
**Steward-Owner is responsible for keeping legal entity running and maintained, thus needs executive voting powers.**

### QUESTION
Should we divide Steward-Ownership up between more than 2 for the start? Should everyone core member have them?

## Definition of Done

- [x] define initial requirements
- [ ] discuss requirements
- [ ] decide on first final iteration of requirements

**Comment by tamagit80** [comment]
Hello, thanks for these precious infos. I would like to understand one important thing. SecretDecks is a tarot project. We said the pilot project. I can't understand why we are using SD as naming for the reputation / autonomus organization system.

I find very hard to explain people what we are doing cause it seems like the tarot project is developing the system. Not easy to understand that the sistem is applicable to different areas.

**Comment by wlstylr** [comment]
@tamagit80 @DanM3rcurius 
I would describe what we are doing as using the Tarot project as a catalyst for forming the SecretDecks DisCo.  Once we have accomplished this task we will be left with DisCo scaffolding that any niche group of creatives could employ to launch their own project.

Eventually would could template and build GUIs around the he DisCo scaffolding to remove technical barriers.

**Comment by DanM3rcurius** [comment]
Yes @wlstylr , the SecretDecks DisCo (or previously called ACO) is needed for the "Tarot project" (I take it you mean the Tarot Metaverse?) to arise. 
The shared ownership, purpose driven legal entity and merit based governance is not possible outside a DisCo framework, as far as I knowingly researched.

**Comment by wlstylr** [comment]
Yes, Tarot project = Tarot Metaverse

**Comment by DanM3rcurius** [comment]
There now will be **voting (A) shares** for both entities:
- SecretDecks 
- DisCoTec

DisCoTec will be an invested member in the SecretDecks cooperative and vice versa; this allows for both to receive interest (via staking rewards) on the funds locked in the shares.

**First I'll setup the SecretDecks cooperative, the charter will be quite similar so once I translated it, most of the work is already done**
